src/payment/payment.ts

```diff
@@ -33,6 +33,8 @@ export interface VaultedInstrument {
     instrumentId: string;
     ccCvv?: string;
     ccNumber?: string;
+    iin?: string;
```

Comment:
I had a quick discussion with @davidchin. This type is public, by adding these properties here, we hint the user they can pass them in, which can cause confusion as we are really not using the copies within the payload.

A better approach will be to create a new interface for the request bigpay, and keep this one free of those properties.

```ts
export type PaymentInstrument = CreditCardInstrument | NonceInstrument | VaultedInstrument | CryptogramInstrument | ThreeDSVaultedInstrument;

// ...

export interface ThreeDSVaultedInstrument extends VaultedInstrument {
    iin?: string;
    threeDSecure?: ThreeDSecure | ThreeDSecureToken;
}
```

Reply:
Thanks heaps for this.